src/app/views/album-detail.html

```diff
@@ -1,14 +1,44 @@
 <div>
-    <h2>EN ESTA SECCION TIENE QUE APARECER LA TAPA DEL DISCO SELECCIONADO JUNTO CON LA LISTA DE TEMAS QUE TIENE EL DISCO.</h2>
-    <img src="https://i.scdn.co/image/66302ca80395b6be3600d5f0ef69db9e0c43f4f5" alt="">
-    <ul>
-        <li>Track 1</li>
-        <li>Track 2</li>
-        <li>Track 3</li>
-        <li>Track 4</li>
-        <li>Track 5</li>
-        <li>Track 6</li>
-        <li>Track 7</li>
-        <li>Track 8</li>
-    </ul>
+    <nav class="nav">
+        <img src="/app/images/spotify-icon.png" alt="" class="logo-left" ng-click=albumDetailCtrl.toIndex()>
```

Comment:
using ng-click here is not accesible, you should wrap the `img` with an `a` tag and use the href to redirect the user to a certain route in the app.